metabrainz/templates/index/privacy.html

```diff
@@ -7,12 +7,144 @@ <h1 class="page-title">{{ _('Privacy Policy') }}</h1>
 
   <h2>{{ _('Summary') }}</h2>
   <p>
-    {{ _('The MetaBrainz Foundation will not collect any private information unless
-    you choose to donate to support the MusicBrainz project. Any personal
-    information you choose to provide during the donation process will not be
-    revealed to anyone else, unless you give us permission to list your name on
-    our donations page. We will not contact you unless you give us permission
-    to do so when you make a donation.') }}
+  <ol>
+    <li>
+      You do not have to provide any personal data to be able to browse the contents of the databases published by MetaBrainz.
+    </li>
+    <li>
+      You do not have to provide any personally-identifying information if you choose not to.
+    </li>
+    <li>
+      We will never reveal your email address on MetaBrainz controlled web sites.
+    </li>
+  </ol>
+  <p>
+    In order to get maximum value out of the MetaBrainz projects, you may want to create an account and log in. 
+    Doing so grants you the ability to: edit the databases, contribute your own data, communicate with other users, and keep track 
+    of recently released music from artists in your MusicBrainz collection. If you choose to create an account then the minimum we 
+    ask is that you: choose a unique username and password, use a web browser that accepts "session" cookies, and provide a verified 
+    email address.
+  </p>
+  <p>
+    If you do not create an account (or are not logged in) you will not have access to the above listed features, however, you will 
+    still have full access to browse and examine the database.
+  </p>
+
+  <h2>{{ _('Applicable to all users') }}</h2>
+
+  <h3>{{ _('Cookies') }}</h3>
+  <p>
+    We use two cookies that contain sensitive information:
+  </p>
+
+  <ol>
+    <li>
+      <b>remember_login</b>: If you choose the "Log in permanently" option when logging in to one of our sites; the cookie will be used to 
+      remember your login details.
+    </li>
+    <li>
+      <b>musicbrainz_server_session</b>: If you are logged into one of our sites, your current session ID is stored in this cookie.
```

Comment:
For AB, CB, and LB the session cookie is called `session` and for BB it is `connect.sid`, so this doesn't seem to be entirely correct.

Maybe make a definition list instead (so multiple "terms" can be listed for one "definition")?

```html
<dl>
  <dt>remember_login</dt>
    <dd>…</dd>
  <dt>musicbrainz_server_session</dt>
  <dt>session</dt>
  <dt>connect.sid</dt>
    <dd>If you are logged into one of our sites, your current session ID will be stored in this cookie (cookie name depends on the sites you're logged in to).</dd>
</dl>
```